static/app/types/hooks.tsx

```diff
@@ -125,6 +136,7 @@ export type CustomizationHooks = {
   'integrations:feature-gates': IntegrationsFeatureGatesHook;
   'member-invite-button:customization': InviteButtonCustomizationHook;
   'member-invite-modal:customization': InviteModalCustomizationHook;
+  'wrapper:make-component-with-analytics': MakeComponentWithAnalyticsHook;
```

Comment:
why can't you just call this like `routes:route-activated` or something, and just have it fire any time a route is rendered?

It can pass like the route path or something, and then in getsentry we can connect the hook to the analytics thing and decide if we want to record page load analytics depending on the route?